modules/ROOT/pages/hybrid-manage-props.adoc

```diff
@@ -10,11 +10,28 @@ image:logo-server-active.png[link="/runtime-manager/deployment-strategies#anypoi
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
 
+
 In an on-premises (hybrid or Anypoint Platform PCE) deployment, you can add properties to an application using the `mule-app.properties` file or Anypoint Runtime Manager.
 
-== Add Application Properties with the mule-app.properties file
+For example, you can define application-specific properties in a properties file, then later add environment-specific properties from Runtime Manager during deployment.
+
+Specifying properties in Runtime Manager enables you to define properties in one place rather than having multiple application property files for a single application in multiple environments.
```

Comment:
This?
```suggestion
Specifying properties in Runtime Manager enables you to define an application's properties in one place rather than in multiple files in multiple environments.